frontend/src/app/App.tsx

```diff
@@ -72,6 +74,29 @@ const App: React.FC = () => {
         </Page>
       );
     }
+    if (appContextStorageClasses.length === 0) {
+      // There was an error fetching cluster storage classes via openshift/dynamic-plugin-sdk-utils
+      return (
+        <Page>
+          <PageSection>
+            <Stack hasGutter>
+              <StackItem>
+                <Alert variant="danger" isInline title="General loading error">
+                  <p>
+                    There was an error fetching the list of cluster StorageClasses via
+                    openshift/dynamic-plugin-sdk-utils
+                  </p>
+                  <p>
+                    Check with your Openshift cluster admins that at least one StorageClass is set
+                    up in the cluster
+                  </p>
+                </Alert>
+              </StackItem>
+            </Stack>
+          </PageSection>
+        </Page>
+      );
+    }
```

Comment:
Is this necessary? This will lock down the app -- not having _any_ storage classes doesn't totally preclude you from flows, it just stops PVC creation. I think we don't want to lock out everything.

Also, we definitely do not want to write comments and/or content text about the `openshift/dynamic-plugin-sdk-utils` as it's not important to the if-statement of "do I have storage classes".

Reply:
agreed, will remove that whole part here, not necessary.